lib/avrora/schema/ocf_mapper.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Avrora.Schema.OCFMapper do
```

Comment:
I think the functionality of this module is shared with `Avrora.Encoder`. I think Encoder should not deal with extracting of OCF if we are going to introduce this module and only this module should actually know how to deal with it and how to craft `Avrora.Schema`

About the naming, I think `Mapper` with `parse` methods is not reflecting what this module is doing. We are not mapping OCF to Schema we are extracting the schema from the OCF format. Hence my suggestions:

1. Rename module to `ObjectContainerFile`
2. Rename function to `extract_schema`

Reply:
Agree with this, so we can have `ObjectContainerFile.extract_map` which will take a payload of ocf message, will look into the memory_storage for the associated schema, if present will resolve automatically otherwise will convert the schema_info decoded by `:avro_ocf.decode_binary(payload)` and will store it in memory_storage

Reply:
Yeah, it sounds like a resolver, but since we can't resolve the ID to Schema, we can extract it with the approach which was used in Resolver 👍